docs/config/amq-broker.spec

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
Is there a reason for this file to be `.spec` instead of `.yml`?

Reply:
I updated the file names to have .yaml extension